### ISSUE TITLE
Some URL segments 404

### ISSUE THREAD

**Issue by mattdistefano** [angular/angular.io#2720]
Right now https://angular.io/docs/ts/latest/api/router/index/ is a valid route but https://angular.io/docs/ts/latest/api/router/ 404s. Not a big deal, but if you're navigating by URL it's a little annoying. Ideally all the segments in the path would have some content or at least a redirect.

**Comment by chalin** [comment]
Right. FYI, a reorganization of the entries below `api/` is planned; and this rework will address the issues you mention.

cc @naomiblack @IgorMinar @alexwolfe @ericjim
